pkg/apis/certmanager/v1alpha1/types_issuer.go

```diff
@@ -172,6 +174,10 @@ type VaultAuth struct {
 	// This Secret contains a AppRole and Secret
 	// +optional
 	AppRole VaultAppRole `json:"appRole,omitempty"`
+
+	// This indicates we wish to use the ServiceAccount token
```

Comment:
Can you expand this description out further? It feels like it is missing the second half of the sentence 🙈